doc/src/guide/listeners.asciidoc

```diff
@@ -328,6 +328,67 @@ processes.
     echo_protocol, []
 ).
 
+=== Setting connection count alarms
+
+The `alarms` transport options allows you to configure alarms
+which will be triggered when the number of connections under a connection
+supervisor reaches or exceeds the defined treshold.
+
+The `alarms` transport option takes a map with alarm names as keys and alarm
+options as values.
+
+Any term is allowed as an alarm name.
+
+Alarm options, defining the alarm behavior, are again a map with the following
+keys, all of which are mandatory:
```

Comment:
The `cooldown` is no longer mandatory.

Reply:
I removed that part of the sentence, put `(5000)` on the description of  `cooldown`, and slightly changed a sentence in the example explanation.